cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -514,3 +514,37 @@ def test_rebuilt_vcenter_duplicate_hosts(appliance, provider):
     # Using appliance.rest_api as hosts.all() do not return archived hosts, I need those too
     hosts_after = len(appliance.rest_api.collections.hosts.all)
     assert hosts_before == hosts_after
+
+
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, override=True)
```

Comment:
Just need ONE here, ONE_PER_TYPE doesn't really apply when you're only collecting one type.